ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/ThreeDSecureRequestedExemptionType.kt

```diff
@@ -0,0 +1,8 @@
+package com.braintreepayments.api.threedsecure
+
+enum class ThreeDSecureRequestedExemptionType(val stringValue: String) {
```

Comment:
I wonder if property `name` for the string value is compatible with the legacy code, maybe there is a chance we don't need addition values.

Reply:
I think the only place where the stringValue is used is where the enum type is converted to json, so there shouldn't be any legacy references to the actual string values. I can double-check to see if we can reduce the visibility modifier of the `stringValue` as merchants wouldn't need to reference it.